project/plugins.sbt

```diff
@@ -4,4 +4,6 @@ resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/release
 
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.15")
 
-addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.34")
\ No newline at end of file
+addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.34")
+
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.15")
```

Comment:
for chart library; Lucian ;   lucidchart -> we can use this brand name in our famous;

Reply:
not sure what you mean :)

Reply:
just an ideea; how to take advantage of faimous brand.   you can call me  Lucian from Lucien Alliance ( SG1- movie)  or Crisan Lucid, like lucidchard